### ISSUE TITLE
freetype_x86 depends on devel:libharfbuzz_x86

### ISSUE THREAD

**Issue by X547** [haikuports/haikuports#5840]
Version: freetype_x86-2.10.4-2-x86_gcc2

Binary packages should not depend on devel packages.

Issue was found in https://review.haiku-os.org/c/haiku/+/3871.

**Comment by X547** [comment]
```
	provides: freetype_x86 = 2.10.4
	provides: lib:libfreetype_x86 = 6.17.4 (compatible >= 6)
	requires: haiku_x86>=r1~beta2_hrev54154_109-1
	requires: lib:libbz2_x86>=1.0.8
	requires: lib:libpng16_x86>=16.37.0
	requires: lib:libz_x86>=1.2.11
	requires: lib:libharfbuzz_x86>=0.20704.0
	requires: devel:libharfbuzz_x86>=0.20704.0 // !!!
```

**Comment by korli** [comment]
Should be fixed with freetype_x86-2.10.4-3. Thanks for the report.
